processors/component/destroy.go

```diff
@@ -67,7 +69,7 @@ func detachNetwork(appModel *models.App, componentModel *models.Component) error
 	display.StartTask("Releasing IPs")
 	defer display.StopTask()
 
```

Comment:
I wonder if there might be a more apparent check here. At first glance I wouldn't know why you're looking for empty IPs. Something like componentModel.State != "active" would be more apparent.